### DISCUSSION TITLE
DIagnostic/Informational Messages

### DISCUSSION THREAD

**Discussion by Phil-Tolbert** [codefori/vscode-ibmi#1083] [Q&A] [answered]
How do I disable the diagnostic/informational messages from showing in a source member when utilizing copy books?
See the example below:
Thanks in advance for any help.
![Capture](https://user-images.githubusercontent.com/102687779/220377801-a58df063-e3b3-4827-b433-26e58c94d723.JPG)

**Comment by worksofliam** [comment, marked as answer]
Add the error code to the 'ignore error' setting in the Connection Settings.

Also, I recommend you turning this on in the VS Code settings:

<img width="934" alt="image" src="https://user-images.githubusercontent.com/3708366/220383346-7d1913d6-e77f-437d-8604-723a9da52343.png">

**Comment by Phil-Tolbert** [comment]
That worked. Thank you very much.
